src/nvim/tui/tui.c

```diff
@@ -1025,161 +1133,1012 @@ static void unibi_set_if_empty(unibi_term *ut, enum unibi_string str,
+  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1,  -1, -72,   3,-118,   4,-112,   4,  27,  91,  90,   0,   7,   0,  13,   0,  27,  91,  37, 105,
+  37, 112,  49,  37, 100,  59,  37, 112,  50,  37, 100, 114,   0,  27,  91,  51, 103,   0,  27,  91,  72,  27,  91,  74,   0,  27,  91,  75,   0,  27,  91,  74,
+   0,  27,  91,  37, 105,  37, 112,  49,  37, 100,  71,   0,  27,  91,  37, 105,  37, 112,  49,  37, 100,  59,  37, 112,  50,  37, 100,  72,   0,  27,  68,   0,
+  27,  91,  72,   0,  27,  91,  63,  50,  53, 108,   0,   8,   0,  27,  91,  63,  50,  53, 104,   0,  27,  91,  67,   0,  27,  77,   0,  27,  91,  80,   0,  27,
+  91,  77,   0,  27,  93,  48,  59,   7,   0,  14,   0,  27,  91,  53, 109,   0,  27,  91,  49, 109,   0,  27,  91,  63,  52,  55, 104,   0,  27,  91,  52, 104,
+   0,  27,  91,  55, 109,   0,  27,  91,  55, 109,   0,  27,  91,  52, 109,   0,  27,  91,  37, 112,  49,  37, 100,  88,   0,  15,   0,  27,  91, 109,  15,   0,
+  27,  91,  50,  74,  27,  91,  63,  52,  55, 108,   0,  27,  91,  52, 108,   0,  27,  91,  50,  55, 109,   0,  27,  91,  50,  52, 109,   0,  27,  91,  63,  53,
+ 104,  36,  60,  49,  48,  48,  47,  62,  27,  91,  63,  53, 108,   0,   7,   0,  27,  55,  27,  91, 114,  27,  91, 109,  27,  91,  63,  55, 104,  27,  91,  63,
+  49,  59,  52,  59,  54, 108,  27,  91,  52, 108,  27,  56,  27,  62,  27,  93,  82,   0,  27,  91,  76,   0, 127,   0,  27,  91,  51, 126,   0,  27,  79,  66,
+   0,  27,  91,  49,  49, 126,   0,  27,  91,  50,  49, 126,   0,  27,  91,  49,  50, 126,   0,  27,  91,  49,  51, 126,   0,  27,  91,  49,  52, 126,   0,  27,
+  91,  49,  53, 126,   0,  27,  91,  49,  55, 126,   0,  27,  91,  49,  56, 126,   0,  27,  91,  49,  57, 126,   0,  27,  91,  50,  48, 126,   0,  27,  91,  49,
+ 126,   0,  27,  91,  50, 126,   0,  27,  79,  68,   0,  27,  91,  54, 126,   0,  27,  91,  53, 126,   0,  27,  79,  67,   0,  27,  91,  66,   0,  27,  91,  65,
+   0,  27,  79,  65,   0,  27,  91,  63,  49, 108,  27,  62,   0,  27,  91,  63,  49, 104,  27,  61,   0,  13,  10,   0,  27,  91,  37, 112,  49,  37, 100,  80,
+   0,  27,  91,  37, 112,  49,  37, 100,  77,   0,  27,  91,  37, 112,  49,  37, 100,  66,   0,  27,  91,  37, 112,  49,  37, 100,  83,   0,  27,  91,  37, 112,
+  49,  37, 100,  76,   0,  27,  91,  37, 112,  49,  37, 100,  68,   0,  27,  91,  37, 112,  49,  37, 100,  67,   0,  27,  91,  37, 112,  49,  37, 100,  84,   0,
+  27,  91,  37, 112,  49,  37, 100,  65,   0,  27,  60,  27,  91,  34, 112,  27,  91,  53,  48,  59,  54,  34, 112,  27,  99,  27,  91,  63,  51, 108,  27,  93,
+  82,  27,  91,  63,  49,  48,  48,  48, 108,   0,  27,  56,   0,  27,  91,  37, 105,  37, 112,  49,  37, 100, 100,   0,  27,  55,   0,  10,   0,  27,  77,   0,
+  27,  91,  48,  37,  63,  37, 112,  49,  37, 112,  54,  37, 124,  37, 116,  59,  49,  37,  59,  37,  63,  37, 112,  50,  37, 116,  59,  52,  37,  59,  37,  63,
+  37, 112,  49,  37, 112,  51,  37, 124,  37, 116,  59,  55,  37,  59,  37,  63,
```

Comment:
If it helps you, I can make the Big Block Of Numbers half as wide, but twice as deep.  (-: